.changeset/gorgeous-coats-shave.md

```diff
@@ -0,0 +1,5 @@
+---
+'apollo-angular': minor
```

Comment:
In an ideal world `apollo-client` would have released a major version, because it was breaking, and we would also have to release a major version. But since they released as patch version, we probably also should release as patch, not minor, shouldn't we ?